python_src/src/lamp_py/ingestion/pipeline.py

```diff
@@ -60,7 +60,7 @@ def main() -> None:
         check_for_sigterm(metadata_queue, rds_process)
         ingest(metadata_queue=metadata_queue)
         check_for_sigterm(metadata_queue, rds_process)
-        time.sleep(60 * 5)
+        time.sleep(5)
```

Comment:
no need to sleep for 5 minutes between event loops.

would also be down to eliminate this all together.